src/wmnds/patterns/cookies/cookies-manager/_cookies-manager.njk

```diff
@@ -0,0 +1,53 @@
+{% macro wmndsCookiesManager(params) %}
+
+{# Imports of components #}
+{% from "wmnds/components/form-elements/checkboxes/_checkboxes.njk" import wmndsCheckboxes %}
+{% from "wmnds/components/button/_button.njk" import wmnButton as primary %}
+
+<div class="wmnds-p-md wmnds-cookies-manager">
+  <div class="wmnds-container wmnds-col-1 wmnds-col-sm-3-4 wmnds-col-lg-3-5">
+    <form class="wmnds-grid wmnds-cookies-manager__form" action="#">
+      <h1 class="wmnds-col-1 wmnds-m-none">Manage Cookies</h1>
+      <p class="wmnds-col-1 wmnds-m-t-md wmnds-m-b-none">
+        Check your settings below and select the cookies you’re happy with.<br /><br />
+        These settings only apply to wmnetwork.co.uk
+      </p>
+      <br /><br />
```

Comment:
Maybe put these sentences in two `<p>` tags rather than using multiple `<br>`s, as the spacing will then be handled by the margins

Reply:
They did that way in the design and it is quite easy to have the correct space between lines that way.
But if you think it's not a valid reason I can change it to <p> and overwrite the margins with the utility classes.